neukinnis/common/models/account.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "account",
+  "base": "PersistedModel",
+  "idInjection": true,
+  "properties": {
+    "name": {
+      "type": "string",
+      "required": true
+    },
+    "account_type": {
```

Comment:
I don't think this is going to give you the desired validation. In common/models/account.js, you need to have something like this: Account.validatesInclusionOf('account_type', {in: ['federal', 'enterprise', 'commercial']});